### PR TITLE
[GEN_BASEADDRESS] Add traces, Reduce width

### PR DESCRIPTION
**Pull request by SergeGautherie**
Minor improvements.

### CODE REVIEW

File: sdk/tools/gen_baseaddress.py [hunk 1]
```diff
@@ -379,7 +387,7 @@ def main():
     if len(dirs) < 1:
         trydir = os.getcwd()
         print(USAGE)
-        print('No path specified, trying the working directory: ', trydir)
+        print('No path specified, trying the working directory:', trydir)
```
Comment (HBelusca) [inline sdk/tools/gen_baseaddress.py:390]: Is the space unnecessary?

Comment (SergeGautherie) [reply sdk/tools/gen_baseaddress.py:390]: Yes: Python (comma) adds one automatically.

File: sdk/tools/gen_baseaddress.py [hunk 2]
```diff
@@ -253,7 +255,10 @@ def gen_baseaddress(self, output_file):
             postfix = ' # should be above 0x%08x' % self.address
         elif self._reserved:
             postfix = ' # reserved'
-        output_file.write('set(baseaddress_%-30s 0x%08x)%s\n' % (name, self.address, postfix))
+        # Current longest name is: 'msvcrt_crt_dll_startup' (22).
+        if len(name) > 22:
+            print('#', name, 'is longer than current width:', len(name), '> 22')
+        output_file.write('set(baseaddress_%-22s 0x%08x)%s\n' % (name, self.address, postfix))
```
Comment (learn-more) [inline sdk/tools/gen_baseaddress.py:261]: What (problem) solves (this) ?
Are you trying to view this file on a monitor with a resolution of 400x300 ?

Comment (SergeGautherie) [reply sdk/tools/gen_baseaddress.py:261]: Trivial space optimization.
Indeed, easier to read with smaller blank in the middle...

File: sdk/tools/gen_baseaddress.py (hunk 1 above)
Comment (HBelusca) [reply sdk/tools/gen_baseaddress.py:390]: ok I see...

File: sdk/tools/gen_baseaddress.py (hunk 2 above)
Comment (learn-more) [reply sdk/tools/gen_baseaddress.py:261]: These are not meant for humans to read them, and even if they were this complicates the code without any added benefit (in my opinion).

**Pull request closed without merging**